docs/source/daml/intro/6_Parties.rst

```diff
@@ -0,0 +1,197 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+6 Parties and Authority
+=======================
+
+DAML is designed for distributed applications involving mutually distrusting parties. In a well constructed contract model, all parties have strong guarantees that nobody cheats or circumvents the rules laid out by templates and choices. In this section you will learn about DAML's authorization rules and how to develop contract models that give all parties the required guarantees. In particular, you'll learn how to
+
+- Pass authority from one contract to another
+- Write advanced choices
+- Reason through DAML's Authorization model
+
+Preventing iou revokation
+-------------------------
+
+The ``SimpleIou`` contract from :doc:`4_Transformations` and :doc:`5_Restrictions` has one major problem: The contract is only signed by the ``issuer``. The signatories are the parties with the power to create and archive contracts. If Alice gave Bob a ``SimpleIou`` for $100 in exchange for some goods, she could just archive it again after receiving the goods. Bob would have a record such actions, but would have to resort to off-ledger means to get his money back.
```

Comment:
```suggestion
The ``SimpleIou`` contract from :doc:`4_Transformations` and :doc:`5_Restrictions` has one major problem: the contract is only signed by the ``issuer``, and the signatories are the parties with the power to create and archive contracts.

This means, if Alice gave Bob a ``SimpleIou`` for $100 in exchange for some goods, she could just archive it again after receiving the goods. Bob would have a record of her actions, but would have to resort to off-ledger means to get his money back.
```

Reply:
I would capitalize after colon if what comes after is a complete sentence. Do you always lower-case after colon?